<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-DC5U0BQ" failures="2" tests="13" name="Default test" time="377.362" errors="0" timestamp="2024-03-15T14:12:43 IST">
  <testcase classname="Page_Test.Login_Page_Test" name="TC001_Verify_Login_with_Invalid_credential" time="8.226"/>
  <testcase classname="Page_Test.Login_Page_Test" name="TC002_invalidmobileno" time="9.866"/>
  <testcase classname="Page_Test.Login_Page_Test" name="TC003_verify_update_number" time="18.056">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.utility.listner.getcapcture(String)&amp;quot; because &amp;quot;com.utility.Base_Utility.lis&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.utility.listner.getcapcture(String)" because "com.utility.Base_Utility.lis" is null
at com.utility.Base_Utility.Custom_click(Base_Utility.java:326)
at Page_Test.Login_Page_Test.TC003_verify_update_number(Login_Page_Test.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC003_verify_update_number -->
  <testcase classname="Page_Test.Login_Page_Test" name="TC004_Verify_Login_with_short_length_of_mobile_number_credential" time="20.154">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.utility.listner.getcapcture(String)&amp;quot; because &amp;quot;com.utility.Base_Utility.lis&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.utility.listner.getcapcture(String)" because "com.utility.Base_Utility.lis" is null
at com.utility.Base_Utility.custom_sendkeys(Base_Utility.java:308)
at Page_Test.Login_Page_Test.TC004_Verify_Login_with_short_length_of_mobile_number_credential(Login_Page_Test.java:66)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC004_Verify_Login_with_short_length_of_mobile_number_credential -->
  <testcase classname="Page_Test.Login_Page_Test" name="TC005_max_digit_mobile_number" time="21.518"/>
  <testcase classname="Page_Test.Login_Page_Test" name="TC006_Terms_of_use_button" time="13.557"/>
  <testcase classname="Page_Test.Login_Page_Test" name="TC007_Privacy_policy" time="29.185"/>
  <testcase classname="Page_Test.Login_Page_Test" name="TC008_contact_Us" time="6.499"/>
  <testcase classname="Page_Test.Login_Page_Test" name="TC009_Continue_as_guest" time="51.86"/>
  <testcase classname="Page_Test.Login_Page_Test" name="TC010_Verify_Login_with_valid_credential" time="38.251"/>
  <testcase classname="Page_Test.Login_Page_Test" name="TC011_resend_butn" time="19.969"/>
  <testcase classname="Page_Test.Login_Page_Test" name="TC012_inavalid_OTP" time="18.953"/>
  <testcase classname="Page_Test.Login_Page_Test" name="TC013_edit_moble" time="39.74"/>
</testsuite> <!-- Default test -->
